src/store/store.ts

```diff
@@ -590,6 +598,7 @@ export const useStore = defineStore('store', {
     isAmalgamationRegularValid (): boolean {
       const isCertifyValid = this.getCertifyState.valid && !!this.getCertifyState.certifiedBy
       const isCourtOrderValid = this.isRoleStaff ? this.getCourtOrderStep.valid : true
+      const isEffectiveDateTimeValid = this.getEffectiveDateTime.valid
```

Comment:
This is an amalgamation bug that I found. The bug is that file and pay would work even if the effective date and time component was invalid (no date was selected). This fixes that.

Reply:
Please mention this to Riyaz to test.

Reply:
Let me fix this in my amalgamation PR so that I let Riyaz know to test it.